<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QDialog" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>861</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>281</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="AdminMenu">
      <widget class="QTableView" name="CampusTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>461</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>120</y>
         <width>170</width>
         <height>108</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>117</y>
         <width>168</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Add New Pending Countries</string>
       </property>
      </widget>
      <widget class="QPushButton" name="revertButton">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>200</y>
         <width>168</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Revert</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AddButton">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>144</y>
         <width>168</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>173</y>
         <width>168</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Revert to Original State</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <property name="accessibleName">
     <string>tab2</string>
    </property>
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QTableView" name="deleteSouvenirTableView">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>371</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="addSouvenirLabel_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Delete Food Item(s)&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="foodbox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteChosenSouvenir">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updatebutton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backBtn_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
